README.md

```diff
@@ -269,6 +283,14 @@ The LIVE indicator only appears during a LIVE streaming video playback. This wil
 
 > [LIVE Indicator Color](https://github.com/aol-public/OneMobileSDK-playground-ios/tree/master/NewTutorials)
 
+#### Animations customization and disabling
```

Comment:
change:
+#### Animations customization and disabling

to:
+#### Animations Customization and Disabling